docs/var.tfvars-doc.md

```diff
@@ -392,3 +392,18 @@ csi_driver_type            = "stable"
 csi_driver_version         = "v0.1.1"
 ```
 **IMPORTANT**: This is an **experimental** feature and not yet ready for production.
+
+These set of variables are specific for LUKS encryption configuration and installation.
+
+```
+luks_compliant              = false # Set it true if you prefer to use FIPS enable in ocp deployment
+luks_config                 = [ { thumbprint = "", url = "" }, { thumbprint = "", url = "" }, { thumbprint = "", url = "" } ]
+luks_filesystem_device      = "/dev/mapper/root"  #Set this value for file system device
+luks_format                 = "xfs"  #Set value of format for filesystem 
+luks_wipeFileSystem         = true  #Set value of wipeFileSystem 
```

Comment:
Could you please do away from camel case? Let us be consistent for all vars and avoid camel case.

Reply:
Updated required changes.